README.md

```diff
@@ -0,0 +1,76 @@
+[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.14186100.svg)](https://doi.org/10.5281/zenodo.14186100)
+
+# NFDI4BIOIMAGE data management illustrations by Henning Falk
+
+These illustrations were contracted by the Heinrich Heine University of Düsseldorf in the frame of the consortium [NFDI4BIOIMAGE](https://nfdi4bioimage.de) from [Henning Falk](https://twitter.com/DrHenningFalk) for the purpose of education and public outreach. The illustrations are free to use under a CC-BY 4.0 license.
+
+NFDI4BIOIMAGE is a consortium of Germany's National Research Data Infrastructure ([Nationale Forschungsdateninfrastruktur, NFDI](https://nfdi.de)).
+NFDI4BIOIMAGE is a collaboration project comprising legally independent partners. The consortium is legally non-independent and does not act autonomously towards third parties
+
+## Attribution
+
+Please include an attribution similar to:
+*["Data annotation matters"](https://github.com/NFDI4BIOIMAGE/Postcards-2024/blob/main/DatasetsAnonymous.jpg), NFDI4BIOIMAGE Consortium (2024): NFDI4BIOIMAGE data management illustrations by [Henning Falk](https://twitter.com/DrHenningFalk), Zenodo, https://doi.org/10.5281/zenodo.14186100, is used under a [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/) license. Modifications to this illustration include cropping.*
+
+## Citing the illustrations
+
+NFDI4BIOIMAGE Consortium (2024): NFDI4BIOIMAGE data management illustrations by Henning Falk, Zenodo, https://doi.org/10.5281/zenodo.14186100, is used under a CC BY 4.0 license (accessed 9.29.24).
+
+## Images by name
+
+- [Data annotation matters](#Data-annoation-matters)
+- [Discovering that your microscopic specimens produce macroscopic data](#Discovering-that-your-microscopic-specimens-produce-macroscopic-data)
+- [Data steward](#Data-steward)
+- [Imaging data life cycle](#Image-data-life-cycle)
+- [Data is available upon request](#Data-is-available-upon-request)
+
+
+### "Data annotation matters"
+
+![Datasets Anonymous](/DatasetsAnonymous.jpg)
```

Comment:
Sometimes these need additional adjustment post-PR. We should re-check that the README renders after merging.